monkeywagtail/artist/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls import url
-from artist.views import ArtistDetailView
+from . import views
```

Comment:
It's recommended by Two Scoops of Django to refer to modules in the same directory. :)